content/applications/finance/sign.rst

```diff
@@ -1,8 +1,24 @@
+:show-content:
+
 ====
 Sign
 ====
 
 .. toctree::
    :titlesonly:
 
-   sign/overview
\ No newline at end of file
+   sign/signature_validity
+
+**Odoo Sign** allows you to send, sign and approve documents online.
+
+You can upload any PDF file and add drag-and-dropping fields on it. These fields are automatically
+filled out with the user's detail if they are logged in.
+
+.. seealso::
+   - `Odoo Sign: product page <https://www.odoo.com/app/sign>`_
+
+Validity of electronic signatures
+=================================
+
+While electronic signatures are usable in most countries, we recommend you check the :doc:`validity
+of electronic signatures <sign/signature_validity>`.
```

Comment:
Until that we add more content to this app's doc, don't you think that its two pages can be safely merged together?